runelite-client/src/main/java/net/runelite/client/plugins/banktags/tabs/TabInterface.java

```diff
@@ -732,8 +724,11 @@ private void deleteTab(String tag)
 		configManager.unsetConfiguration(CONFIG_GROUP, ICON_SEARCH + tag);
 		tabManager.save();
 
-		updateBounds();
-		scrollTab(0);
+		clientThread.invoke(() ->
```

Comment:
You already have clientThread.invoke in place where you call deteleTab, why not just put entire deleteTab call into clientThread? And remove this invoke here

Reply:
Yeah, and in both cases it is from the ChatBoxPanelManager. Imo it would be better to wrap it in ClientThread there